### ISSUE TITLE
Pacman- SSL certificate problem.

### ISSUE THREAD

**Issue by HieuLe-88** [msys2/MSYS2-packages#2529]
## Describe the issue

SSL certificate problem.

## Steps to Reproduce the Problem
When I execute "pacman -Syu", it shows alot of errors related to SSL certificate problem. How can I solve this issue?
1. Download MSYS2 here https://www.msys2.org/
2. Install it
3. Execute bellow command.
>pacman -Syu
4. Errors:
:: Synchronizing package databases...
error: failed retrieving file 'mingw32.db' from repo.msys2.org : SSL certificate problem: self signed certificate in certificate chain
error: failed retrieving file 'mingw32.db' from downloads.sourceforge.net : SSL certificate problem: self signed certificate in certificate chain
error: failed retrieving file 'mingw32.db' from www2.futureware.at : SSL certificate problem: self signed certificate in certificate chain
error: failed retrieving file 'mingw32.db' from mirror.yandex.ru : SSL certificate problem: self signed certificate in certificate chain
error: failed retrieving file 'mingw32.db' from mirrors.tuna.tsinghua.edu.cn : SSL certificate problem: self signed certificate in certificate chain

**Comment by HieuLe-88** [comment]
Solved:
Solution:
Copy certificate of https://www.msys2.org/ to ca-bundle.crt file in C:\msys64\usr\ssl\certs

**Comment by Biswa96** [comment]
In my setup, those are already in that path.

**Comment by SaihttamR** [comment]
> Solved: Solution: Copy certificate of https://www.msys2.org/ to ca-bundle.crt file in C:\msys64\usr\ssl\certs

I copied the cert to the according folder but I still get the cert error. 
Would you provide me how to export the cert from www.msys2.org? Maybe i did something wrong...

**Comment by Biswa96** [comment]
Is it not required to copy certificates manually. Try to update the packages with `pacman -Syyu` twice or use new installer from the website.

**Comment by RzNmKX** [comment]
and what about those of us behind corporate proxies/firewalls that have their own SSL certificates?

**Comment by Biswa96** [comment]
> what about those of us behind corporate proxies/firewalls that have their own SSL certificates/perform MITM?

The procedure is explained here https://www.msys2.org/docs/faq/

**Comment by vinukshetty** [comment]
Most of them still face this issue even with the above solution, if you are also one among them then try this. 

**Solution:**
Export certificate from https://repo.msys2.org/ and rename *.goskope.com to ca-bundle.crt file and place it into C:\msys64\usr\ssl\certs as shown below. 

Note: please click on the highlighted hierarchy while downloading the cert file.

![Capture2](https://github.com/msys2/MSYS2-packages/assets/8382327/8d82a81d-9505-4127-bfc7-a06fdc0bfa58)
![image](https://github.com/msys2/MSYS2-packages/assets/8382327/b5cc1764-3a86-43e0-8332-05c617fea1ca)

**Comment by the-moog** [comment]
The instructions given by @vinukshetty did not work exactly as given for myself, using Firefox/Win11.
This is what I had to do....

1. Click the padlock on the address bar for https://repo.msys2.org/
2. Select Connection Secure -> More Information
3. On the "Page Info" dialog, click "View Certificate"
4. On the "About Certificate" page I see three different subjects, all three have a **_Miscellaneous_** section.
5. There are "Download Certificate" buttons PEM (cert) | PEN (chain)
6. Click the PEM (chain)
7. Next, I used WSL to copy and rename the certificates (msys is installed in C:\msys64\)
```bash
cd /mnt/c/Users/the-moog/Downloads
mv /mnt/c/msys64/usr/ssl/certs/ca-bundle.crt /mnt/c/msys64/usr/ssl/certs/ca-bundle.crt.orig
cp repo-msys2-org-chain.pem /mnt/c/msys64/usr/ssl/certs/ca-bundle.crt
```
